cas-server-core/src/main/java/org/jasig/cas/services/DefaultRegisteredServiceAccessStrategy.java

```diff
@@ -60,6 +62,12 @@
     /** Is the service allowed to use SSO? **/
     private boolean ssoEnabled = true;
 
+    private String startingDateTime;
```

Comment:
Shouldn't we have a `TimeBasedRegisteredServiceAccessStrategy` for this need in addition to the `DefaultRegisteredServiceAccessStrategy`?